sdk-swagger/src/main/java/com/ifountain/opsgenie/client/swagger/ApiClient.java

```diff
@@ -618,15 +627,15 @@ private ClientResponse getAPIResponse(String path, String method, List<Pair> que
     ClientResponse response = null;
 
     if ("GET".equals(method)) {
-      response = (ClientResponse) builder.get(ClientResponse.class);
+      response = builder.get(ClientResponse.class);
     } else if ("POST".equals(method)) {
-      response = builder.type(contentType).post(ClientResponse.class, serialize(body, contentType, formParams));
+      response = builder.buildPost(Entity.json((String) serialize(body, contentType, formParams))).invoke(ClientResponse.class);
```

Comment:
`(String)` type conversion is incorrect here and in the all cases below. `serialize()` returns `CreateAlertRequest` for `application/json` content type.